public/coffee/ide.coffee

```diff
@@ -127,6 +127,13 @@ define [
 		ide.permissionsManager = new PermissionsManager(ide, $scope)
 		ide.binaryFilesManager = new BinaryFilesManager(ide, $scope)
 
+		# Set up labels
+		$scope.$on 'doc:labels:updated', labels.onDocLabelsUpdated
+		$scope.$on 'entity:deleted', labels.onEntityDeleted
+		$scope.$on 'file:upload:complete', labels.fileUploadComplete
+		$timeout () ->
+			labels.loadProjectLabelsFromServer()
+
```

Comment:
How about we move this into a `LabelsManager`, like we have with the few lines above, each part of the editor is in it's own little wrapper. So have a folder structure like:

```
coffee/ide/labels/
    LabelsManager.coffee
    services/
        labels.coffee
```

to keep it all in one place.